client/src/client.rs

```diff
@@ -60,15 +60,15 @@ impl Into<OutPoint> for JsonOutPoint {
 }
 
 /// Shorthand for converting a variable into a serde_json::Value.
-fn into_json<T>(val: T) -> Result<serde_json::Value>
+pub fn into_json<T>(val: T) -> Result<serde_json::Value>
```

Comment:
Isn't this outside the crate the same than `serde_json::to_value()`?

Reply:
Yes, it is. But I think if you use shorthand versions in methods realization - it would be great to provide them to easily replace and write custom realizations of methods. Because it's a little tediously to copy/paste them in a project folder.